src/v3/src/components/StepperRadio/StepperRadio.tsx

```diff
@@ -71,11 +72,12 @@ const StepperRadio: UISchemaElementComponent<{
 
   return (
     <FormControl component="fieldset">
-      {label && (<FormLabel>{label}</FormLabel>)}
+      {label && (<FormLabel id={labelId}>{label}</FormLabel>)}
       <RadioGroup
         name={name}
         value={value as string}
         onChange={handleChange}
+        aria-labelledby={labelId}
```

Comment:
Should this also be conditional to match cases when the label is not provided for a radio group? (Eg. Security questiion enrollment form does not have a label for the radio group).

Reply:
This is for "select a security question" vs "custom security question"? Where the values are "self-evident"?

Reply:
`src/v3/test/integration/__snapshots__/authenticator-enroll-security-question.test.tsx.snap` was updated.